b2share/modules/b2share_demo/__init__.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of EUDAT B2Share.
+# Copyright (C) 2016 CERN.
+#
+# B2Share is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# B2Share is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with B2Share; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""B2Share demonstration."""
+
+from __future__ import absolute_import, print_function
+
+from .ext import B2ShareDemo
+
+__all__ = (
+    'B2ShareDemo',
+)
```

Comment:
b2share_demo was a separate package in V2 codebase, but it caused troubles when loading resources from it.
For this reason b2share_demo was changed to a module instead.
The module can be easily excluded from final B2SHARE package (or `pip install -e .`) by not defining b2share_demo in entrypoints.